packages/executors/src/serverExecutor.ts

```diff
@@ -71,20 +72,18 @@ export class ServerExecutor implements IExecutor {
     this.token = this.retrieveSSEToken();
   }
 
-  private initializeSocket() {
+  private async initializeSocket() {
     if (!this.sandboxId) {
       throw new Error('initializeSocket: sandboxId is not defined');
     }
-
     const usedHost = this.host || 'https://codesandbox.io';
-    const sseHost = usedHost.replace('https://', 'https://sse.');
+    const sseLbHost = usedHost.replace('https://', 'https://sse-lb.');
+    const res = await axios.get(`${sseLbHost}/api/cluster/${this.sandboxId}`);
+    const sseHost = res.data.hostname;
 
     this.socket = io(sseHost, {
       autoConnect: false,
       transports: ['websocket', 'polling'],
-      query: {
-        sandboxid: this.sandboxId,
```

Comment:
Do we lose any data in the manager with this? Might there be a reason in the future that we want to infer the sandbox id from the WS connection? If that's the case we might want to keep it in the URL, but I'm not sure if you already have this data.

Reply:
The client already sends the sandbox id to `sse-manager` in the websocket protocol.